packages/create-react-app/index.js

```diff
@@ -30,7 +30,7 @@
 // tell people to update their global version of create-react-app.
 //
 // Also be careful with new language features.
-// This file must work on Node 0.10+.
+// This file must work on Node 4+.
```

Comment:
This is intentional. It shouldn't crash on old Nodes because we want to be able to print an error message explaining we only support Node 4 before crashing. Otherwise the error is cryptic.